drydock_backups/plugin.py

```diff
@@ -0,0 +1,237 @@
+from __future__ import annotations
+
+import os
+import os.path
+from glob import glob
+
+import click
+import pkg_resources
+from tutor import hooks
+
+from .__about__ import __version__
+
+config = {
+    "defaults": {
+        "VERSION": __version__,
+        "IMAGE": "ednxops/shipyard-utils",
```

Comment:
Does this image have some level of versioning? (It should BTW)

Reply:
@Henrrypg if versioning was added, please update the default value

Reply:
On CI we are using the time and the commit hash, but there is no currently other versioning system